postgres/problems/problem_04.md

```
@@ -1,20 +1,23 @@
-# 04 ユーザ毎の注文回数
+# 04 ユーザ名の重複排除一覧
 
 **難易度**：★  
-**学習トピック**：GROUP BY, 集約
+**学習トピック**：SELECT, DISTINCT
 
 ## 説明
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)